[2025-02-13T21:39:20.485+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T21:39:20.500+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-07-27T13:17:00+00:00 [queued]>
[2025-02-13T21:39:20.508+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-07-27T13:17:00+00:00 [queued]>
[2025-02-13T21:39:20.509+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-13T21:39:20.520+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2024-07-27 13:17:00+00:00
[2025-02-13T21:39:20.526+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=14681) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T21:39:20.527+0000] {standard_task_runner.py:72} INFO - Started process 14686 to run task
[2025-02-13T21:39:20.526+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2024-07-27T13:17:00+00:00', '--job-id', '20711', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpi07pqolc']
[2025-02-13T21:39:20.528+0000] {standard_task_runner.py:105} INFO - Job 20711: Subtask fetch_and_store_crypto_data
[2025-02-13T21:39:20.556+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-07-27T13:17:00+00:00 [running]> on host 7e1fe653b817
[2025-02-13T21:39:20.647+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-27T13:17:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-27T13:17:00+00:00'
[2025-02-13T21:39:20.647+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T21:39:20.693+0000] {logging_mixin.py:190} INFO - Erreur lors de la récupération des données CoinGecko : 429 Client Error: Too Many Requests for url: https://api.coingecko.com/api/v3/simple/price?ids=bitcoin%2Cethereum&vs_currencies=usd
[2025-02-13T21:39:20.694+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-13T21:39:20.699+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T21:39:20.700+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2024-07-27T13:17:00+00:00, execution_date=20240727T131700, start_date=20250213T213920, end_date=20250213T213920
[2025-02-13T21:39:20.740+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T21:39:20.775+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-13T21:39:20.778+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
